cytotable/utils.py

```diff
@@ -149,6 +149,93 @@ def _duckdb_reader() -> duckdb.DuckDBPyConnection:
     )
 
 
+def _sqlite_mixed_type_query_to_parquet(
+    source_path: str,
+    table_name: str,
+    chunk_size: int,
+    offset: int,
+    result_filepath: str,
+) -> str:
+    """
+    Performs SQLite table data extraction where one or many
+    columns include data values of potentially mismatched type
+    such that the data may be exported to Arrow and a Parquet file.
+
+    Args:
+        source_path: str:
+            A str which is a path to a SQLite database file
+        table_name: str:
+            The name of the table being queried
+        chunk_size: int:
+            Row count to use for chunked output
+        offset: int:
+            The offset for chunking the data from source.
+        dest_path: str:
+            Path to store the output data.
```

Comment:
This is a very minor nitpick, but some argument descriptions end in a period and some don't. I don't have a preference between the two, but I think it'd be better if they were consistent.

Reply:
Good point, thanks, I'll update these!